README.md

```diff
@@ -42,68 +63,138 @@ Tested with both SPT 3.8.3 and SPT 3.9.x and the associated Fika versions.
+
+   - **Retrieve Profile ID**: The profile ID is the filename (excluding the `.json` extension) of the profile generated in the server's `user/profiles` directory. For example, if the profile is named `670c0b1a00014a7192a983f9.json`, the profile ID is `670c0b1a00014a7192a983f9`.
+
+4. **Configure the Dedicated Client**
+
+   - **Edit Fika Core Configuration**: Open the `com.fika.core.cfg` file located in the dedicated client's `BepInEx/config/` directory.
+
+     - Update values for `Force Bind IP` and/or `Force IP`. It might be sufficient to set `Force Bind IP` to `Disabled`, and to set `Force IP` to the IP of the host interface.
+       If you are running a VPN, then this is your VPN IP.
+
+       ```
+       ## Force Bind IP
+       # Set to Disabled
+       Force Bind IP = Disabled
+
+       ## Force IP
+       # Set to the IP address of your host interface (e.g., your LAN IP or VPN IP)
+       Force IP = your.host.interface.ip
+       ```
+
+5. **Run the Docker Image**
+
+   - **Mount the Fika Client Directory**: Ensure that your dedicated client installation directory is mounted to `/opt/tarkov` in the container. If you are running the dedicated client on a server, you will need to copy and transfer the dedicated client installation to the server (~40GB).
+
+   - **Set Environment Variables**: When running the docker image, set the following environment variables:
+
+     - `PROFILE_ID` to the profile ID obtained in step 3.
+
+     - `SERVER_URL` to your server's URL or IP address.
+
+     - `SERVER_PORT` to your server's port (usually `6969`).
+
+     - Optionally, set `USE_MODSYNC` to `true` if you are using [Corter-ModSync](https://github.com/c-orter/modsync/) for plugin synchronization.
+
+   - **Run the Docker Container**:
+
+     ```shell
+     docker run --name fika_dedicated \
+       -v /path/to/fika:/opt/tarkov \
+       -e PROFILE_ID=blah \
+       -e SERVER_URL=your.spt.server.ip \
+       -e SERVER_PORT=6969 \
+       -p 25565:25565/udp \
+       ghcr.io/zhliau/fika-headless-docker:latest
+     ```
+
+     With docker-compose file:
+     ```yaml
+     services:
+       fika_dedicated:
+         image: ghcr.io/zhliau/fika-headless-docker:latest
+         container_name: fika_dedi
+         volumes:
+           - /host/path/to/fika:/opt/tarkov
+         environment:
+           - PROFILE_ID=adadadadadadaadadadad
+           - SERVER_URL=your.spt.server.ip
+           - SERVER_PORT=6969
+           - USE_MODSYNC=true # If you want to use modsync on this dedicated client
+         ports:
+           - 25565:25565/udp
+     ```
+
+     If you are running the SPT server in docker, you can make use of docker-compose's network DNS to connect the dedicated client to the SPT server:
```

Comment:
I know I originally wrote this part of the line, but while you're in here, I think this would be better as 
```suggestion
     If you are running the SPT server as a service in the same docker-compose stack, you can make use of docker-compose's network DNS to resolve the SPT server from the dedicated client:
```